test/import/FSHImporter.Profile.test.ts

```diff
@@ -1211,6 +1211,27 @@ describe('FSHImporter', () => {
         );
       });
 
+      it('should parse fixed value using Canonical', () => {
+        const input = `
+        CodeSystem: Example
+        * #first
+        * #second
+
+        Profile: ObservationProfile
+        Parent: Observation
+        * code.coding.system = Canonical(Example)
+        `;
+
+        const result = importSingleText(input);
+        const profile = result.profiles.get('ObservationProfile');
+        expect(profile.rules).toHaveLength(1);
+
+        const expectedCanonical = new FshCanonical('Example')
+          .withLocation([8, 32, 8, 49])
+          .withFile('');
+        assertFixedValueRule(profile.rules[0], 'code.coding.system', expectedCanonical);
+      });
+
```

Comment:
Can you add a similar test but use the optional spaces (e.g., `Canonical (   Example   )`)?  I think that the resulting `FshCanonical` should have the spaces trimmed out (else it doesn't export correctly).